### PR TITLE
Kern import fix

### PR DESCRIPTION
**Pull request by manoskary**
This PR addresses issues #158 and #159 and other minor bug fixes.

### CODE REVIEW

**Review by sildater — APPROVED** [review]

File: partitura/io/importmusicxml.py [hunk 1]
```diff
@@ -347,6 +347,26 @@ def _parse_parts(document, part_dict):
...
+            if o.end is None:
+                # if len(o.start.starting_objects[score.Repeat]) > 0:
+                #     starting = list(o.start.starting_objects[score.Repeat].keys())[0]
+                #     # if unstarted repeat from volta, continue for now
+                #     if len(starting.end.ending_objects[score.Repeat]) > 0:
+                #         # if repeat from volta, continue for now
+                #         volta_repeats.append(o)
+                #         continue
+
+                end_times = [r.start.t for r in part.iter_all(score.Repeat)] + [part._points[-1].t]
```
Comment (sildater) [inline partitura/io/importmusicxml.py:362]: r.start.t might not exist for all repeats (which was, for r.end.t in line 382, the error that made this change necessary), so maybe there should be an additional safety?

**Review by sildater — CHANGES_REQUESTED** [review]

File: partitura/io/importmusicxml.py (hunk 1 above)
Comment (manoskary) [reply partitura/io/importmusicxml.py:362]: Although the `r.start.t` might have caused the initial error, the correct version of this score was only possible by adding the inverse loop of assuming repeat ends.  I don't know if those two loop cases might cause any other security threats.

Would you have any proposal for this??

Comment (manoskary) [reply partitura/io/importmusicxml.py:362]: For context and for other possible reviewers/readers of this thread. The "bug" that was corrected concerns scores that have a repeat start at some position of the score but at end/Fine has no repeat signs, so when partitura musicxml parser searches how to close the repeat start with another `pt.score.Repeat` element it fails.

Comment (sildater) [reply partitura/io/importmusicxml.py]: I added a safety in 005aba513427c7ea423a9cfc6a57c2ba55784e30

Comment (sildater) [reply partitura/io/importmusicxml.py]: it's just so end_times doesn't loop over *all* repeats, but only well-positioned ones.

Comment (sildater) [reply partitura/io/importmusicxml.py]: (it might work in a single list comprehension, I don't know, I'm editing in the browser.)

**Review by sildater — APPROVED** [review]

**Pull request merged**